R/test_functions.R

```diff
@@ -0,0 +1,36 @@
+#######################
+# TEST nc_make_network
```

Comment:
Unit testing should be done through the `tests/testthat` folder and the [`testthat`](http://testthat.r-lib.org/) framework.